jsk_perception/sample/sample_darknet_yolo_object_detection.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
```

Comment:
https://github.com/jsk-ros-pkg/jsk_recognition/tree/master/jsk_perception/sample

I mean most of the sample launch files are named as `sample_<node_name>.launch` and I'd like make this file also follow that rule.